src/ui-core/components/molecules/ProductCard/ProductCard.component.tsx

```diff
@@ -1,16 +1,8 @@
 import React from "react";
-import "./ProductCard.css";
-import "../../App.css";
-import { Category } from "../../enums/category.ts";
-
-interface ProductCardProps {
-  title: string;
-  image: string;
-  price: number;
-  description: string;
-  category: Category.MensClothing | Category.WomensClothing;
-  descriptionBackgroundColor: string;
-}
+import "./ProductCard.component.css";
+import "../../../../App.css";
```

Comment:
@N-i-z You can import this `App.css` with common styles at global level—let's add it to `App.tsx` file & remove it in component level...